IFC4x3/Properties/e/Emodulus_10zwR$ze1ElBZzkStdNfro/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_10zwRze1ElBZzkStdNfro" Name="Emodulus" UniqueId="40f7a6ff-f680-4ebc-b8fd-b9cde75e9d72" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```

Comment:
Reuse "YoungModulus".

Reply:
Young modulus is for tensile strength right? Ok to reuse if this is valid here also. Then we should change at a couple of other places as well. @j-weil.

Reply:
Fixed in [de5d9ff](https://github.com/bSI-InfraRoom/IFC-Specification/pull/627/commits/de5d9ff1a04f75256973e43b7d4b9cd6ea662e68)